src/Sylius/Bundle/ChannelBundle/composer.json

```
@@ -64,7 +64,6 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Sylius\\Bundle\\ChannelBundle\\spec\\": "spec/",
             "Sylius\\Bundle\\ChannelBundle\\Tests\\": "Tests/"
```

You could change it to be consistent with other packages:
```suggestion
            "Tests\\Sylius\\Bundle\\ChannelBundle\\": "Tests/"
```